AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/Keyrings/AwsKms/AwsKmsHierarchicalKeyring.dfy

```diff
@@ -609,15 +619,15 @@ module AwsKmsHierarchicalKeyring {
     const cryptoPrimitives: Primitives.AtomicPrimitivesClient
     const branchKeyId: string
     const ttlSeconds: Types.PositiveLong
-    const cache: SynchronizedLocalCMC.SynchronizedLocalCMC
+    const cache: StormTrackingCMC.StormTrackingCMC
```

Comment:
Does the keyring care that it is a `StormTrackingCMC`? Or does it really just want the interface?